specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/examples/CanonicalSupportPlanTypes_Get.json

```diff
@@ -9,7 +9,7 @@
       "body": [
         {
           "enabled": true,
-          "supportPlanType": "standard",
+          "supportPlanType": "standad",
```

Comment:
is this a real fix? or a spell error in swagger?

Reply:
To test the spellcheck output. Will revert back later.